src/content/learn/index.md

```diff
@@ -1,45 +1,45 @@
 ---
-title: Quick Start
+title: Mulai Cepat
 ---
 
 <Intro>
 
-Welcome to the React documentation! This page will give you an introduction to the 80% of React concepts that you will use on a daily basis.
+Selamat datang di dokumentasi React! Halaman ini akan memberikan Anda pengenalan tentang 80% konsep React yang akan Anda gunakan sehari-hari.
 
 </Intro>
 
 <YouWillLearn>
 
-- How to create and nest components
-- How to add markup and styles
-- How to display data
-- How to render conditions and lists
-- How to respond to events and update the screen
-- How to share data between components
+- Cara membuat dan menyarangkan (*nest*) komponen-komponen
+- Cara menambahkan *markup* dan penggayaan (*styles*)
+- Cara menampilkan data
+- Cara me-*render* kondisi dan daftar (*lists*)
+- Cara merespons *events* dan memperbarui layar
+- Cara berbagi data antar komponen
 
 </YouWillLearn>
 
-## Creating and nesting components {/*components*/}
+## Membuat dan menyarangkan (nesting) komponen {/*components*/}
 
-React apps are made out of *components*. A component is a piece of the UI (user interface) that has its own logic and appearance. A component can be as small as a button, or as large as an entire page.
+Aplikasi React dibuat dari *komponen*. Komponen adalah bagian dari UI (*user interface*, antarmuka pengguna) yang memiliki logika dan tampilan tersendiri. Sebuah komponen dapat berukuran sekecil tombol, atau sebesar seluruh halaman.
 
-React components are JavaScript functions that return markup:
+Komponen React adalah fungsi JavaScript yang mengembalikan *markup*:
 
 ```js
 function MyButton() {
   return (
-    <button>I'm a button</button>
+    <button>Saya adalah tombol</button>
   );
 }
 ```
 
-Now that you've declared `MyButton`, you can nest it into another component:
+Sekarang setelah Anda mendeklarasikan `MyButton`, Anda dapat menyarangkannya dengan komponen lain:
 
 ```js {5}
 export default function MyApp() {
   return (
     <div>
-      <h1>Welcome to my app</h1>
+      <h1>Selamat datang di aplikasi saya</h1>
```

Comment:
`app` should be translated into `aplikasi`?

I think I haven't seen any convention for this on other pages.

Reply:
Yes, correct. @mhaidarhanif